src/components/chat/__tests__/IncomingResponseMessage.test.tsx

```diff
@@ -54,25 +54,28 @@ describe('IncomingResponseMessage', function () {
     expect(screen.getByText('SecondColumn')).toBeInTheDocument();
   });
 
-  test('should render proportion as percentage and with two decimal places', function () {
-    const message: IncomingResponseMessageProps = {
-      message: {
-        role: 'GenSpectrum',
-        textBeforeData: 'textBeforeData',
-        data: [{ proportion: 0.12345 }],
-        textAfterData: 'textAfterData',
-      },
-      toBeLogged: true,
-      onRateUp: jest.fn(),
-      onRateDown: jest.fn(),
-      onComment: jest.fn(),
-    };
-
-    renderIncomingResponseMessage(message);
-
-    expect(screen.getByText('textBeforeData')).toBeInTheDocument();
-    expect(screen.getByText('textAfterData')).toBeInTheDocument();
-    expect(screen.getByText('Proportion (%)')).toBeInTheDocument();
-    expect(screen.getByText('12.35')).toBeInTheDocument();
-  });
+  // The IncomeResponseMessage component – along with the entire ChatPage – is currently not under use, so
```

Comment:
so does this mean that the ChatPage will stop working with this update or that the tests are broken? Does the ChatPage still exist on cov-spectrum? Maybe we want to delete it first to not have a broken page

Reply:
The code here is unused as there is no actual chat feature at the moment but instead we show at https://cov-spectrum.org/chat this:

<img width="898" height="654" alt="image" src="https://github.com/user-attachments/assets/f67dda01-a5e5-4822-8a76-1fc44b5ceaa1" />

I created an issue to remove the code (#1128).